### PR TITLE
Adding kinect_autonomy to documentation index for distro

### PR DESCRIPTION
**Pull request by TheD00MSlayer**
I'd like my repo to be indexed and documented on ros.org.

### CODE REVIEW

File: indigo/distribution.yaml [hunk 1]
```diff
@@ -5889,6 +5889,14 @@ repositories:
       url: https://github.com/mrpt-ros-pkg/kinect_2d_scanner.git
       version: master
     status: maintained
+  kinect_autonomy:
+    doc:
+        type: python
+        url: https://github.com/GunnerPatterson/ROS-Kinect-Autonomy
+        version: indigo
+        depends:
+        -openni
```
Comment (tfoote) [inline indigo/distribution.yaml]: You need a space after the `-`s

File: indigo/distribution.yaml [hunk 2]
```diff
@@ -5889,6 +5889,14 @@ repositories:
       url: https://github.com/mrpt-ros-pkg/kinect_2d_scanner.git
       version: master
     status: maintained
+  kinect_autonomy:
+    doc:
+        type: python
+        url: https://github.com/GunnerPatterson/ROS-Kinect-Autonomy
+        version: indigo
+        depends:
```
Comment (tfoote) [inline indigo/distribution.yaml]: This key is out of alphabetical order, it should be first.

File: indigo/distribution.yaml [hunk 3]
```diff
@@ -5889,6 +5889,14 @@ repositories:
       url: https://github.com/mrpt-ros-pkg/kinect_2d_scanner.git
       version: master
     status: maintained
+  kinect_autonomy:
+    doc:
+        type: python
+        url: https://github.com/GunnerPatterson/ROS-Kinect-Autonomy
```
Comment (tfoote) [inline indigo/distribution.yaml]: The url needs to end in `.git`

File: indigo/distribution.yaml [hunk 4]
```diff
@@ -5889,6 +5889,14 @@ repositories:
...
+        version: indigo
```
Comment (tfoote) [inline indigo/distribution.yaml]: There's no `indigo` branch only a `master` branch.

**Review by tfoote — CHANGES_REQUESTED** [review]
There's some formatting issues with this that I commented on inline. However at bigger issue is that I looked into the repository and saw that the repository does not have any ROS packages in it. For the indexing to work it needs to have the formal structure of a ROS package such as a `package.xml` to be able to generate the metadata for indexing. http://wiki.ros.org/ROS/Tutorials/CreatingPackage

**Comment by mikaelarguedas** [comment]
@GunnerPatterson friendly ping

**Comment by mikaelarguedas** [comment]
Closing due to no response. @GunnerPatterson feel free to open a new PR once the comments have been addressed.

**Pull request closed without merging**